.eslintrc.json

```diff
@@ -26,6 +26,9 @@
     "react/forbid-prop-types": 0,
     "import/prefer-default-export": 0,
     "class-methods-use-this": 0,
-    "no-plusplus": 0
+    "no-plusplus": 0,
+    "no-trailing-spaces": 0,
+    "no-prototype-builtins": 0,
+    "arrow-parens": 0
```

Comment:
i know this rule is a little annoying, but it helps to avoids errors like the following one, can u please re-enable it?:
/*eslint-env es6*/
```
// Bad
if (a => 2) {
}

// Good
if (a >= 2) {
}
```

Reply:
Which rule? `no-plusplus`?

Reply:
`arrow-parens`

Reply:
Thanks! I will enable it!